index.json

```diff
@@ -60,13 +61,5 @@
       { "extensions": [".js", ".jsx"] }
     ],
     "react/prefer-stateless-function": "off"
-  },
-  "plugins": ["flowtype", "import", "promise", "compat", "react"],
-  "settings": {
-    "import/resolver": {
-      "webpack": {
-        "config": "configs/webpack.config.eslint.js"
```

Comment:
I'm assuming you changed this because relative path will change when `index.json` is in `node_modules`. Is that correct?